tensorflow/lite/micro/kernels/cmsis-nn/fully_connected.cc

```diff
@@ -65,24 +47,11 @@ TfLiteStatus CalculateOpData(TfLiteContext* context,
                              const TfLiteTensor* filter,
```

Comment:
CalculateOpData() can be removed if we propagate "data->buffer_idx = -1;" into Prepare().

Reply:
done.